repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -706,7 +707,7 @@ export const getPassengerTypeByNameAndNocCode = async (
 
 interface SavedPassengerType {
     group: GroupPassengerType;
-    single: PassengerType;
+    single: ActualPassengerType;
```

Comment:
Could this be named 'SinglePassengerType'? 'Actual' isn't massively clear to me what it means

Reply:
That is a lovely suggestion, I went with `actual` because my mind went totally blank. Thanks, I'll update.